providers/rethinkdb.js

```diff
@@ -14,6 +14,14 @@ module.exports = class extends Provider {
 		return this.db;
 	}
 
+	/**
+	 * Pings the Rethinkdb server.
+	 * @returns {number}
+	 */
+	async ping() {
+		return new Date() - await this.db.now();
```

Comment:
`new Date()` should be `Date.now()`

Reply:
this could probably be 
`return this.db.now().then(ping => Date.now() - ping);` and remove async

Reply:
In a perfect world, you'd save the date.now before returning. *hint HINT*